### PR TITLE
Supports Avalonia 0.10

### PR DESCRIPTION
**Pull request by rstm-sf**
Hello!

Supports Avalonia 0.10

### CODE REVIEW

File: src/Factory.cs [hunk 1]
```diff
@@ -16,8 +16,9 @@ public static FormattedText CreateText(MsaglDrawing.Label label)
       var ftext = new FormattedText
       {
         Constraint = new Size(10, 10),
-        Typeface = new Typeface(fontFamily, fontSize, fontStyle, fontWeight),
-        Text = label.Text
+        Typeface = FontManager.Current.GetOrAddTypeface(fontFamily, fontStyle, fontWeight),
```
Comment (rstm-sf) [inline src/Factory.cs:19]: See https://github.com/AvaloniaUI/Avalonia/pull/4627

File: sample/MainWindow.xaml [hunk 2]
```diff
@@ -2,8 +2,8 @@
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" 
         xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
-        xmlns:agc="using:AvaloniaGraphControl" 
-        xmlns:local="using:AvaloniaGraphControlSample"
+        xmlns:agc="clr-namespace:AvaloniaGraphControl;assembly=AvaloniaGraphControl" 
+        xmlns:local="clr-namespace:AvaloniaGraphControlSample;assembly=AvaloniaGraphControlSample"
```
Comment (rstm-sf) [inline sample/MainWindow.xaml:6]: Because https://github.com/AvaloniaUI/Avalonia/issues/5167

**Comment by slateAGF** [comment]
Hi. Are there any lingering issues with the support for Avalonia 0.10 release?
Thank you for your work

**Comment by Oaz** [comment]
The only reason for not merging is that I was on other stuff and I haven't used Avalonia for a while.

**Pull request merged**